nixos/modules/services/misc/linux-enable-ir-emitter.nix

```diff
@@ -0,0 +1,62 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+with lib; let
+  cfg = config.services.linux-enable-ir-emitter;
+in
+{
+  options = {
+    services.linux-enable-ir-emitter = {
+      enable = mkEnableOption (mdDoc "") // {
+        description = mdDoc ''
+          Whether to enable IR emitter hardware. Designed to be used with the
+          Howdy facial authentication. After enabling the service, configure
+          the emitter with `sudo linux-enable-ir-emitter configure`.
+        '';
+      };
```

Comment:
why the description is added like this is beyond me, regardless  `lib.mdDoc` is is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557
this goes for every instance but I wont comment on each.

Reply:
It was a suggestion from long ago. I can revert to using `mkEnableOption "IR emitter hardware. ...".

Reply:
looking good, neat solution.